composer.json

```diff
@@ -18,7 +18,7 @@
     "require": {
         "php": "^7.2||^8.0",
         "doctrine/annotations": "^1.13 || ^2.0",
-        "doctrine/instantiator": "^1.0.3",
+        "doctrine/instantiator": "^1.0.3|^2.0",
```

Comment:
```suggestion
        "doctrine/instantiator": "^1.0.3 || ^2.0",
```

Reply:
Is still valid syntax?

Reply:
Yes, both `|` and `||` are valid ([but according to the docs, the latter is the preferred variant](https://getcomposer.org/doc/articles/versions.md#version-range)). Also, everywhere else in the `require` section the `||` syntax is used, and I think we should prefer consistency.